components/BestListForm.tsx

```diff
@@ -0,0 +1,126 @@
+import Image from "next/image";
+import styles from "./BestListForm.module.css";
+import medalIcon from "../public/medal_icon.svg";
+import heart from "../public/heart.svg";
+import { useEffect, useState } from "react";
+import { getArticles } from "../lib/api";
+
+interface Writer {
+  id: number;
+  nickname: string;
+}
+interface Article {
+  content: string;
+  createdAt: string;
+  id: number;
+  image: string | null;
+  likeCount: number;
+  title: string;
+  writer: Writer;
+}
+interface ArticleList {
+  list: Article[];
+}
+
+interface BestProps {
+  pageSize: number;
+  orderBy: string;
+}
+
+const PC_List = 3;
+const TABLET_List = 2;
+const MOBILE_List = 1;
+
+export default function BestListForm() {
+  const [orderBy, setOrderBy] = useState("like");
+  const [isLike, setIsLike] = useState<ArticleList>({ list: [] });
+  const [pageSize, setPageSize] = useState<number>(PC_List);
+
+  async function getBestArticle({ pageSize, orderBy }: BestProps) {
```

Comment:
getBestArticle는 컴포넌트가 아닌 함수이므로 Params라는 명칭을 사용하는 것이 좋아보입니다.

```suggestion
  async function getBestArticle({ pageSize, orderBy }: getBestArticleParams) {
```